### PR TITLE
Feat/recaps

### PR DESCRIPTION
**Pull request by mcclowes**
# Description

Trying to make the ‘using the api’ section clearer, and trying consistently including a recap and read next section.

### CODE REVIEW

File: docs/using-the-api/authentication.md [hunk 1]
```diff
@@ -11,19 +11,38 @@ You must keep the API key secret, so make sure it isn't available in publicly ac
 
 Codat expects the API key to be included in all API requests to the server, Base64 encoded within an 'Authorization' header.
 
-`Authorization: Basic YOUR_ENCODED_API_KEY`
+`Authorization: Basic <YOUR_ENCODED_API_KEY>`
 
 :::info
-You must replace <code>YOUR_ENCODED_API_KEY</code> with your API key, Base64 encoded
+Replace <code>\<YOUR_ENCODED_API_KEY\></code> with your API key, Base64 encoded
```
Comment (d-laing) [inline docs/using-the-api/authentication.md:17]: The backslash is being rendered in the preview build.

Comment (d-laing) [reply docs/using-the-api/authentication.md:17]: Google's style guidelines on [using placeholders](https://developers.google.com/style/placeholders) is to use all caps and underscores, with no angled brackets. 

Elsewhere in the docs, we use lowercase text surrounded by angled brackets. On balance, I think that is the best way to go for our docs.

File: docs/using-the-api/authentication.md [hunk 2]
```diff
@@ -11,19 +11,38 @@ You must keep the API key secret, so make sure it isn't available in publicly ac
...
-2. In the **API Keys** section, copy your authorization header rather than the API key itself.
+2. Copy your authorization header from the relevant table column (rather than the API key itself).
+
+## Managing keys
+
+You can have up to 10 keys per client. You can create new keys in the Portal at **Developers > API keys**.
+
+You can delete keys in the Portal too. Deleted keys will no longer authorize API calls.
+
+:::info Recap
```
Comment (d-laing) [inline docs/using-the-api/authentication.md:37]: The recap is great and something we can replicate. However I don't think the exclamation mark in the admonition looks right here - it makes me think something has gone wrong!

image

File: docs/using-the-api/managing-companies.md [hunk 3]
```diff
@@ -5,14 +5,21 @@ createdAt: "2022-11-07T19:57:50.555Z"
 updatedAt: "2022-12-21T06:24:11.564Z"
 ---
 
-## User onboarding
+## Onboarding your users
 
-Use the following process to onboard users to the Codat API. You can either onboard users:
+Your users/customers are [companies](/core-concepts/companies). To access their data you'll need to onboard them.
```
Comment (d-laing) [inline docs/using-the-api/managing-companies.md:10]: Suggestion:

Your users or customers are represented as companies.

File: docs/using-the-api/managing-companies.md [hunk 4]
```diff
@@ -5,14 +5,21 @@ createdAt: "2022-11-07T19:57:50.555Z"
...
+
+You'll have to go through the following steps:
```
Comment (d-laing) [inline docs/using-the-api/managing-companies.md:12]:
```suggestion
To onboard a new user or customer:
```

File: docs/using-the-api/managing-companies.md [hunk 5]
```diff
@@ -5,14 +5,21 @@ createdAt: "2022-11-07T19:57:50.555Z"
...
+1. Create a company
+2. Authorize access to sources of data
+3. Pull the data
+
+You can either onboard users:
 
 - When they first create an account with your service
-- At the first point you wish to create a connection to their accounting data
+- At the first point you wish to retrieve their financial data
```
Comment (d-laing) [inline docs/using-the-api/managing-companies.md:20]:
```suggestion
- At the first point you want to retrieve their financial data
```

File: docs/using-the-api/managing-companies.md [hunk 6]
```diff
@@ -42,7 +49,9 @@ The name of the company doesn't have to be unique. It's just there to help you i
 The `id` property that you receive in the response is the unique Codat identifier for this company. We recommend that you retain it for future reference.
 :::
 
-## Redirect the user
+### Authorize access
+
+Once you've created the company, they'll need to give you permission to pull their data. There are several approaches to this, but for simplicity we've just cover our out-of-the-box [hosted link](/auth-flow/authorize-hosted-link) approach.
```
Comment (d-laing) [inline docs/using-the-api/managing-companies.md:54]:
```suggestion
Once you've created the company, they'll need to give you permission to pull their data. There are several approaches to doing this, but for simplicity we've just covered our out-of-the-box [hosted link](/auth-flow/authorize-hosted-link) approach.
```

File: docs/using-the-api/managing-companies.md [hunk 7]
```diff
@@ -133,3 +144,20 @@ Creating a new company may cause additional data to be pulled from the platform
 Get a `redirect` URL for the company by following the process [here](/auth-flow/authorize-hosted-link). Send the user to the `redirect` URL. They will be prompted to select their accounting software and complete the linking process using the [Link flow](/auth-flow/overview).
 
 Once the user finishes the Link flow, they will be redirected back to the Redirect URL, as described [earlier in this guide](/using-the-api/managing-companies#redirect-the-user). At this point the re-authorization process is complete and their data has begun synchronizing again.
+
+## Deleting companies
+
+You can delete companies in the Portal in the table in **Companes**.
```
Comment (d-laing) [inline docs/using-the-api/managing-companies.md:150]:
```suggestion
You can delete companies in the Portal in the table in **Companies**.
```

File: docs/using-the-api/modified-dates.md [hunk 8]
```diff
@@ -66,3 +66,15 @@ GET /companies/{companyId}/data/invoices?page=1&query=issueDate%3C{todayMinus12M
 ```
 
 This query retrieves invoices with issue dates that are greater than twelve months old and a source modified date that is greater than or equal to one month old.
+
+:::info Recap
+You have learned:
```
Comment (d-laing) [inline docs/using-the-api/modified-dates.md:71]:
```suggestion
You've learned:
```

File: docs/using-the-api/modified-dates.md [hunk 9]
```diff
@@ -66,3 +66,15 @@ GET /companies/{companyId}/data/invoices?page=1&query=issueDate%3C{todayMinus12M
...
+- How dates are used to keen company data up to date
```
Comment (d-laing) [inline docs/using-the-api/modified-dates.md:72]:
```suggestion
- How dates are used to keep company data up to date
```

File: docs/using-the-api/queueing-data-syncs.md [hunk 10]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Queueing data syncs"
+title: "Refreshing company data"
```
Comment (d-laing) [inline docs/using-the-api/queueing-data-syncs.md:2]: Great change and aligns with the UI.

**Review by d-laing — CHANGES_REQUESTED** [review]
Great changes. I've left a few comments.

File: docs/using-the-api/authentication.md (hunk 1 above)
Comment (mcclowes) [reply docs/using-the-api/authentication.md:17]: Happy to go with google's.

Could you add that link to the the contributing section? We should build up a documented style guide

File: docs/using-the-api/authentication.md (hunk 2 above)
Comment (mcclowes) [reply docs/using-the-api/authentication.md:37]: Tip?

https://docusaurus.io/docs/markdown-features/admonitions

Comment (pzaichkina) [reply docs/using-the-api/authentication.md:37]: I tend to do tips for recaps (e.g. first steps guide)

**Review by d-laing — APPROVED** [review]
All looks good.

**Pull request merged**